Sources/AirbnbSwiftFormatTool/airbnb.swiftformat

```diff
@@ -1,13 +1,10 @@
 # Current (adjusted) version of formatting rules used at Airbnb: 
 # https://github.com/airbnb/swift
-# Requires this version of SwiftFormat, the most recent version 
-# which supports both this ruleset and our deployment target:
-# https://github.com/calda/SwiftFormat/releases/tag/0.51-beta-1
 
 --exclude **/Pods,**/fastlane,**/Submodules
 
 # options
---swiftversion 5.6 # Airbnb is using Swift 5.7, but we aren't ready to yet. This should be incremented when we start using Xcode 14.x.
+--swiftversion 5.8 # This change is provisional, and may need to be decremented when we roll it out
 --self remove # redundantSelf
```

Comment:
Maybe we can do this, but I won't know until I run it through some testing. I feel OK about having the upgrade in the rules fork with a note that it may not be permanent; it indicates to me that it is the target.